src/main/webapp/list.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
```

Comment:
(couldn't figure out how to make a file comment)

Can we rename this file to be a bit more descriptive?   maybe at the very least "list_state.html" or "state_selection.html" to narrow down what the purpose of this webpage is :)